legacy_publisher/dita_publisher.py

```diff
@@ -1,5 +1,71 @@
+import json
+import random
+import os 
 
-def publish_regions(map_url, regions, target_folder):
-    print('outputting regions', map_url, target_folder)
-    for reg in regions:
-        print("outputting region", reg.shape, reg.url, reg.coords)
\ No newline at end of file
+from xml.dom import minidom
+from os.path import dirname, abspath
+from legacyman_parser.utils.constants import DITA_REGIONS_EXPORT_FILE
```

Comment:
there is just one file called `regions.dita`  It will always be called `regions.dita`.  We can hard-code the issue.